### ISSUE TITLE
Hi, I have some question about intent hope you could reply me.

### ISSUE THREAD

**Issue by iamzww** [applenob/RNN-for-Joint-NLU#3]
1.is there an attention on Intent Detection task in your code? you seem to just make a full-connected layer after Bi-LSTM. That confused me.  
2.why  the final intent vector is encoder_final_state_h rather than summation of all encoder_state_h_i?

**Comment by applenob** [comment]
Yes you are right. I didn't use attention in the classification model. I would add this part or you can commit your code if you like. Thank you for pointing this out.
